tests/integration/targets/meraki_ms_link_aggregation/meraki_network/tasks/main.yml

```diff
@@ -200,171 +212,177 @@
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkCombined
-      enable_my_meraki: yes
-      enable_remote_status_page: no
+      local_status_page_enabled: true
+      remote_status_page_enabled: false
     delegate_to: localhost
     register: status_exclusivity
-    ignore_errors: yes
+    failed_when: false
+    changed_when: false
 
-  - assert:
+  - name: Assert status_exclusivity
+    ansible.builtin.assert:
       that:
         - '"must be true when setting" in status_exclusivity.msg'
 
   - name: Create network with one tag
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkTag
       type: switch
       timezone: America/Chicago
       tags: first_tag
     delegate_to: localhost
     register: create_net_tag
-    
+
   - name: Create network with two tags
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkTags
       type: switch
       timezone: America/Chicago
-      tags: 
+      tags:
         - first_tag
         - second_tag
     delegate_to: localhost
     register: create_net_tags
 
-  - set_fact:
-      tag_net_id: '{{create_net_tags.data.id}}'
+  - name: Set fact tag_net_id
+    ansible.builtin.set_fact:
+      tag_net_id: '{{ create_net_tags.data.id }}'
 
   - name: Modify network by net_id
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
-      net_id: '{{tag_net_id}}'
+      org_name: '{{ test_org_name }}'
+      net_id: '{{ tag_net_id }}'
       type: switch
       timezone: America/Chicago
-      tags: 
+      tags:
         - first_tag
         - second_tag
         - third_tag
     delegate_to: localhost
     register: create_net_modified
 
   - name: Modify network with idempotency
-    meraki_network:
+    cisco.meraki.meraki_network:
       auth_key: '{{ auth_key }}'
       state: present
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkTags
       type: switch
       timezone: America/Chicago
-      tags: 
+      tags:
         - first_tag
         - second_tag
         - third_tag
     delegate_to: localhost
     register: create_net_modified_idempotent
 
-  - assert:
+  - name: Assert create_net_modified_idempotent
+    ansible.builtin.assert:
       that:
         - create_net_modified_idempotent.data is defined
 
   - name: Present assertions
-    assert:
+    ansible.builtin.assert:
       that:
         - create_net_combined.data.type == 'combined'
-        - create_net_combined.data.disable_my_meraki_com == True
```

Comment:
Why are we removing these lines?

Reply:
Due to that feature has been removed in the commit https://github.com/CiscoDevNet/ansible-meraki/commit/a6a6d4